### DISCUSSION TITLE
Is drazzy.com down?

### DISCUSSION THREAD

**Discussion by plasticbrain777** [SpenceKonde/megaTinyCore#967] [Q&A] [answered]
I'm getting nowhere trying to install megaTinyCore into Arduino IDE V2.1.0.  As far as I can tell drazzy.com is not responding.  Doesn't respond to ping and Arduino IDE is unable to establish a connection...

Some indexes could not be updated. Get "http://drazzy.com/package_drazzy.com_index.json": read tcp 192.168.1.26:57866->52.4.232.83:80: wsarecv: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond.: http://drazzy.com/package_drazzy.com_index.json

**Comment by hmeijdam** [comment]
I think it is.
When I start the boards manager (V1.8.19) I also get an error.
`Error downloading http://drazzy.com/package_drazzy.com_index.json
`

**Comment by SpenceKonde** [comment, marked as answer]
Yes. The server, hosted in the Amazon AWS cloud decided to put a bullet through it's virtual harddrive, Corruption all over the place, the web root directory was turned into a file instead of a directory, etc. So I need to completely rebuild the webserver in on a new instance

**Comment by SpenceKonde** [comment]
So far I'm on the 4th try deploying the server because I can't get the private key file to be accepted and you can't change that without blowing away instance and redeploying.

**Reply by SpenceKonde** [reply]
5 now, that time I forgot to create a new keypair

I stil do not uinderstand why my old keypair doesn't work,.,,,,

**Comment by SpenceKonde** [comment]
should be up now
